src/ui/Networking.qml

```diff
@@ -83,12 +83,36 @@ Page {
 
     } // Frame
 
+
     // Roles: ip, port, source, block, lastSeenIn, lastSeenOut
     // Implement the model in the backend (a more recommendable approach)
-    ListModel { // EXAMPLE
+    //ListModel { // EXAMPLE
```

Comment:
does this need to be removed ?

Reply:
yes, already removed, this was the QML initial examples